template/{{.input_root_dir}}/{{template `project_name_alphanumeric_underscore` .}}/databricks.yml.tmpl

```diff
@@ -27,6 +27,7 @@ include:
 # Deployment Target specific values for workspace
 targets:
   dev: {{ if (eq .input_include_models_in_unity_catalog `yes`)}} # UC Catalog Name {{ end }}
+    mode: development
```

Comment:
@arpitjasa-db Was this intentional?

We have an integration test that confirms successful template initialization and noticed that the job names now are prefixed with the username (DAB default for `mode: development`), so checking.

Reply:
I intentionally added it as I thought it would be a best practice and most environment I've been working with would benefit from it (as multiple developers could be deploying the same project into dev).
But if that has any negative impact the change could be reverted

Reply:
It is, no need to revert! It seemed like a piggybacked change, so want to make sure there isn't any docs that need to be updated, or otherwise other downstream changes that need to be made.

Reply:
Yup intentional! Didn't realize this would break the naming downstream, thanks for flagging. For docs though, we don't really mention the file names for the dev target so we should be good on that

Reply:
Thanks for confirming.